### DISCUSSION TITLE
Bring dialogue into focus, and frequency of words statistic

### DISCUSSION THREAD

**Discussion by mzsoli** [vkbo/novelWriter#1546] [Ideas]
Two things came to mind while using this great program:

1.) Would it be feasible to have an on/off button to bring dialogue into focus, highlight it with a different color, or mask non-dialogue text? 
(it would facilitate: checking the rhythm of the text, checking its naturalness, or the subtle give-and-take between characters, the flow of the text)

2.) Would it be possible to have statistics on the frequency of words used? (To avoid repetitions of words.)

I would like to take this opportunity to say that I think this programme is simply wonderful. I've tried a lot of novel-writing software, but this is amazingly usable, clean and easy to use. Thank you for creating it. :-)

**Comment by vkbo** [comment]
Thanks! I'm glad it is useful to you 😃 

1. I'm not sure what you mean. There is already a feature to highlight dialogue. It should be on by default. You can change the settings in [Preferences > Highlighting] if it isn't.

![image](https://github.com/vkbo/novelWriter/assets/1619840/12e058fa-b3c7-4d9f-8556-ad7e4574018d)

2. Possibly. I'm considering adding text analysis tools to the Manuscript Build Tool. You can make a feature request for it, and I'll add it to the list of features in #1269. These will only be run on what is visible in the preview, which would make more sense anyway as this is filtered on the text you want to include in the final manuscript.

**Reply by mzsoli** [reply]
Oh, thank you.
I think I know why I slipped through the cracks.
Because I'm writing in Hungarian, and in Hungarian books they use a dash for dialogue. That's what caused the confusion. 

I think the text analysis tools are a very good idea. :smile: 

Thank you very much for the info

**Reply by vkbo** [reply]
I could add a dialogue highlight feature for lines starting with certain characters, if that helps. The dash is sometimes used in Norwegian for dialogue too.

**Reply by vkbo** [reply]
Moved out of thread reply by @mzsoli (see #1393):

> Oooooh that would be great. 🙏
> I was just thinking of writing a scene at most and letting a search for the dash,
> and replace it with (" ").
